### PR TITLE
tc-build: Add s390 support

### PR DESCRIPTION
**Pull request by nathanchance**

### CODE REVIEW

File: kernel/build.sh [hunk 1]
```diff
@@ -27,6 +27,8 @@ while (( ${#} )); do
                     "AArch64") TARGETS=( "${TARGETS[@]}" "aarch64-linux-gnu" ) ;;
                     "ARM") TARGETS=( "${TARGETS[@]}" "arm-linux-gnueabi" ) ;;
                     "PowerPC") TARGETS=( "${TARGETS[@]}" "powerpc-linux-gnu" "powerpc64-linux-gnu" "powerpc64le-linux-gnu" ) ;;
+                    # SystemZ is consumed until Linux 5.6
```
Comment (msfjarvis) [inline kernel/build.sh]: Should this also wait until then?

Comment (nathanchance) [reply kernel/build.sh]: We can build a mainline kernel just fine with latest tip of tree. I just do not really want to add a prerelease kernel to this script (but we could when 5.6-rc1 drops). I do not have a strong opinion.

**Comment by nathanchance** [comment]
I adjusted the help text in the LLVM script, hence the force push.

**Comment by nathanchance** [comment]
I'll respin this once Linux 5.6 is out since this has sat for that long.

**Pull request closed without merging**